### DISCUSSION TITLE
How to make facet plot data don't have time dimension ?

### DISCUSSION THREAD

**Discussion by hafez-ahmad** [pydata/xarray#5204] [Q&A]
I have downloaded sst monthly data of 2009 from https://oceandata.sci.gsfc.nasa.gov/. the dataset doesn't time dimension. I have tried to time dimension manually. My way is not working.

`xr.open_dataset('sst1.nc').assign_coords({'time': pd.date_range(start='2020/01/01',end='2020/1/31',freq='M')})`
![image](https://user-images.githubusercontent.com/20365917/115630029-2d0ee380-a325-11eb-9f61-8745b3130ecf.png)

I have tried, I don't know I have tried and this is not working.do I need to add the time dimension in another way? 

I have attached my dataset here.
https://www.dropbox.com/s/ch4l9c6063rjgv7/bob.zip?dl=0

Your assistant will be highly appreciated.
Thank you
Hafez

**Comment by dcherian** [comment]
Hi @hafez-ahmad the variable `chlor_a` only has `lat, lon` dimensions so it is a single timestep. You probably have to download one file per month
